apps/src/aichat/types.ts

```diff
@@ -33,6 +35,10 @@ export interface Notification extends ChatEvent {
   id: number;
   text: string;
   notificationType: 'error' | 'success';
+  // This field is optional but when it is defined, it must be set to `true`.
+  // This allows the notification to not be included in chat history as default
+  // without having to add an extra field.
+  includeInChatHistory?: true;
```

Comment:
We're using a type of `true` here but allowing a `boolean` when creating an error notification in `dispatchSaveFailNotification`, right? Would be good to be consistent either way - in `onSaveFail`, we are using `flaggedProperties` to determine whether `includeInChatHistory` is true or false, so it seems like we do want to allow this value to be either `true` or `false`, I think?

Reply:
Oh yes! I first defined this, then went back and added the check for `flaggedProperties` on a later update so will update to `boolean`. Thanks!